### ISSUE TITLE
Allow setting description for connection fields

### ISSUE THREAD

**Issue by brandondoran** [graphql-nexus/schema#576]
Currently there does not seem to be a way to specify a `description` for a field created with the `connectionField` plugin.

It seems that you should be able to do something like:

```javascript
export const ThingsQuery = extendType({
  type: 'Query',
  definition(t) {
    t.connectionField('things', {
      type: 'Thing',
      description: 'Returns a list of things',
      nodes: (_root, _args, ctx) => ctx.db.findAllThings()
    });
  }
});
```

which would result the following in generated SDL: 

```graphql
type Query {
  """
  Returns a list of things
  """
  things(...)
}
```

This issue seems similar to #512

**Comment by tgriesser** [comment]
Just merged this in #578 - should be out with a release soon

**Comment by brandondoran** [comment]
Awesome, thanks for the quick action!

**Comment by tgriesser** [comment]
Released in [0.17.0](https://github.com/graphql-nexus/schema/releases/tag/v0.17.0)
